### PR TITLE
ci: Add setup scripts for experimental kernel

### LINKED ISSUE #1961: setup scripts for kernel experimental

### PR DESCRIPTION
**Pull request by GabyCT**
This will allow us to build and install the experimental kernel
for the virtiofs 3.0

Fixes #1961

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: .ci/install_kata_kernel.sh [hunk 1]
```diff
@@ -30,6 +30,8 @@ kernel_repo_dir="${GOPATH}/src/${kernel_repo}"
 kernel_arch="$(arch)"
 readonly tmp_dir="$(mktemp -d -t install-kata-XXXXXXXXXXX)"
 packaged_kernel="kata-linux-container"
+#Experimental kernel support. Pull from virtio-fs GitLab instead of kernel.org
+experimental_kernel="false"
```
Comment (chavafg) [inline .ci/install_kata_kernel.sh]: `experimental_kernel="${experimental_kernel:-false}"` so that the value can be changed using envars?

Comment (GabyCT) [reply .ci/install_kata_kernel.sh]: @chavafg changes applied

File: .ci/install_kata_kernel.sh [hunk 2]
```diff
@@ -105,15 +120,19 @@ main() {
...
+			if ! install_prebuilt_kernel; then
+				info "failed to install cached kernel, trying to build from source"
+				build_and_install_kernel
+			fi
+		else
 			build_and_install_kernel
 		fi
-
-	else
+	fi
```
Comment (devimc) [inline .ci/install_kata_kernel.sh]: `else` didn't work ?

Comment (GabyCT) [reply .ci/install_kata_kernel.sh]: @chavafg changes applied

Comment (GabyCT) [reply .ci/install_kata_kernel.sh]: @devimc changes applied

**Pull request merged**